### PR TITLE
Adding API endpoints

### PR DESCRIPTION
**Pull request by TOLoneWolf**
Added The following Endpoints:
GET	https://addons-ecs.forgesvc.net/api/v2/minecraft/version
GET	https://addons-ecs.forgesvc.net/api/v2/minecraft/version/timestamp
GET	https://addons-ecs.forgesvc.net/api/v2/minecraft/modloader
GET	https://addons-ecs.forgesvc.net/api/v2/minecraft/modloader/timestamp
GET	https://addons-ecs.forgesvc.net/api/v2/minecraft/modloader/{VersionName}
GET	https://addons-ecs.forgesvc.net/api/v2/category
GET	https://addons-ecs.forgesvc.net/api/v2/category/timestamp
GET	https://addons-ecs.forgesvc.net/api/v2/game
GET	https://addons-ecs.forgesvc.net/api/v2/game/timestamp

I haven't used "API Blueprint" format before so I hope I have everything correctly formatted. Wasn't sure if tab/spacing matters.
Didn't see an easily accessible editor/Validator.

### CODE REVIEW

**Comment by skyecodes** [comment]
There's also :
GET https://addons-ecs.forgesvc.net/api/v2/minecraft/version/{versionString}

File: apiary.apib [hunk 1]
```diff
@@ -1675,3 +1687,792 @@ If you would like to submit any changes please make a pull request at https://gi
...
+		"slug": "mc-mods",
+		"avatarUrl": "https://media.forgecdn.net/avatars/52/101/636111139584399357.png",
+		"dateModified": "2018-08-08T19:42:39.42Z",
+		"parentGameCategoryId": null,
+		"rootGameCategoryId": null,
+		"gameId": 432
+	  }
+	]
+
+## Get Category Info [/api/v2/v2/category/{Categoryid}]
```
Comment (skyecodes) [inline apiary.apib]:
```suggestion
## Get Category Info [/api/v2/category/{Categoryid}]
```

File: apiary.apib [hunk 2]
```diff
@@ -1675,3 +1687,792 @@ If you would like to submit any changes please make a pull request at https://gi
...
+		"supportsPartnerAddons": false,
+		"supportedClientConfiguration": 3,
+		"supportsNotifications": true,
+		"profilerAddonId": 43270,
+		"twitchGameId": 18122,
+		"clientGameSettingsId": 1
+	  }
+	]
+
+## Get Game Info [/api/v2/v2/game/{GameID}]
```
Comment (skyecodes) [inline apiary.apib]:
```suggestion
## Get Game Info [/api/v2/game/{GameID}]
```

**Review by skyecodes — CHANGES_REQUESTED** [review]
There are a few typos left.

**Comment by TOLoneWolf** [comment]
Thanks Franckyi for pointing those out.

I noticed some of the parameters where not getting highlighted right (using sublimetext3), (integer) is the type currently there but in the documentation for API Blueprint it shows (number) as the correct type.
https://apiblueprint.org/documentation/specification.html
Just want to make sure that is correct before changing those since I am new to the API Blueprint formatting and not sure if apiary follows that specification.

**Comment by Gaz492** [comment]
A simple way of checking if something is correct is to use https://apiary.io

**Pull request merged**